app/models/server_role.rb

```diff
@@ -5,32 +5,19 @@ class ServerRole < ActiveRecord::Base
   validates_presence_of     :name
   validates_uniqueness_of   :name
 
-  FIXTURE_DIR = File.join(Rails.root, "db/fixtures")
-
-  def self.seed_data
-    File.read(File.join(FIXTURE_DIR, "#{to_s.pluralize.underscore}.csv"))
-  end
-
   def self.seed
-    data = seed_data.split("\n")
-    cols = data.shift.split(",")
-
-    data.each do |a|
-      next if a =~ /^#.*$/ # skip commented lines
+    CSV.foreach(fixture_path, :headers => true, :skip_lines => /^#/).each do |csv_row|
+      action = csv_row.to_hash
+      action['license_required'] ||= ''
```

Comment:
@chessbyte I deleted the column, but I think there are some leftovers:

1. A bunch of `.po` files have [lines](https://github.com/DNNX/manageiq/blob/server-role/config/locales/ja/manageiq.po#L12314-L12315) like below:
```po
msgid "ServerRole|License required"
msgstr "ライセンスが必要"
```
2. There is also [config/model_attributes.rb](https://github.com/ManageIQ/manageiq/blob/master/config/model_attributes.rb#L2024).
```ruby
_('ServerRole|License required')
```

In the first case I can go and remove these lines manually. `model_attributes.rb`, however, is autogenerated and its first and last lines say that it should not be changed manually. How do I regenerate it?